godtools/App/Features/SetupParallelLanguage/Views/ParallelLanguageModal.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  ParallelLanguageModal.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 12/10/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+
+class ParallelLanguageModal: UIViewController {
```

Comment:
Hey @reldredge71 I'm wondering if we can do something different for this view.  To me it just doesn't look that great, too small and hard to navigate.

I was thinking, it would be nice to have a language picker view that can be shared because in the future there will be a capability for picking a language once inside of a tract.  It would be nice to present this view from a flow and have a callback with the picked language and the flow can do whatever it needs to from there with the picked language.  My thoughts for this view is having a full screen list of languages with search bar at the top for filtering.  Pretty much similar to the choose language view.  What are your thoughts on that?

![language-picker](https://user-images.githubusercontent.com/59846460/147112996-2322fc09-8131-41ed-91c2-24fa091a500d.png)

Reply:
That would be nice, and I could re-use the screens we already have rather than making something new.  But that is up to Ann and Justin.  The Figma design shows a different interface.